RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -80,9 +85,12 @@ struct PurchaseInformation {
                         : .expired
                     return ExpirationOrRenewal(label: label, date: .date(dateString))
                 }
+                self.isLifetimeSubscription = expiresDate == nil
```

Comment:
The transaction type is subscription, so this should always be false since it will never be a lifetime purchase